contracts/BaseLaunchpeg.sol

```diff
@@ -181,7 +181,7 @@ abstract contract BaseLaunchpeg is
         override
         onlyOwner
     {
-        if (_feePercent > BASIS_POINT_PRECISION / 5) {
+        if (_feePercent > BASIS_POINT_PRECISION / 4) {
```

Comment:
Change `BASIS_POINT_PRECISION / 4` to `2500` and then add a comment that the max is 25%